scripts/bump-spin-formula.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+VERSION=$1
+FORMULA=${2:-Formula/Spin.rb}
+
+usage() {
+    echo "Usage: $0 <VERSION> [<FORMULA_PATH>]"
+    echo "Updates the Spin Formula for the specified release version"
+    echo "Example: $0 3.0.0"
+}
+
+if [ $# -ne 1 ]; then
+    usage
+    exit 1
+fi
+
+# Get the checksum file for the release
+wget -qO checksums.txt "https://github.com/fermyon/spin/releases/download/$VERSION/checksums-$VERSION.txt"
+
+# Update the formula version
+ERSION="${VERSION:1}"
```

Comment:
```suggestion
VERSION="${VERSION:1}"
```

Or actually `ERSION` is a clever name now that I see what this is doing, chopping off the first character (assumed to be 'v') here... but maybe rendered moot depending on outcome of my other comments.